component_config/configRowSchema.json

```diff
@@ -1 +1,175 @@
+            "prefill": 100,
+            "propertyOrder": 2,
+            "description": "Maximum volume of reviews data to extract per place. Use 0 for metadata-only extraction, or 99999 for comprehensive data ingestion of all available reviews."
+        },
+        "reviewsSort": {
+            "enum": [
+                "newest",
+                "mostRelevant",
+                "highestRanking",
+                "lowestRanking"
+            ],
+            "type": "string",
+            "title": "Sort reviews by",
+            "default": "newest",
+            "enumTitles": [
+                "Newest",
+                "Most relevant",
+                "Highest ranking",
+                "Lowest ranking"
+            ],
+            "propertyOrder": 3,
+            "description": "Define the sorting algorithm for review data extraction to prioritize the most relevant data for your analytics needs."
+        },
+        "reviewsOrigin": {
+            "enum": [
+                "all",
+                "google"
+            ],
+            "type": "string",
+            "title": "Reviews origin",
+            "default": "all",
+            "propertyOrder": 4,
+            "description": "Filter data source to optimize pipeline for specific analytics needs - extract all available reviews or focus exclusively on Google-verified content."
+        },
+        "reviewsStartDate": {
+            "type": "string",
+            "title": "Only scrape reviews newer than [date]",
+            "propertyOrder": 5,
+            "description": "Configure incremental loading by specifying a date threshold. Use either absolute date (e.g. `2024-05-03`) or relative date from now into the past (e.g. `8 days`, `3 months`). JSON input also supports adding time in both absolute (ISO standard, e.g. `2024-05-03T20:00:00`) and relative (e.g. `3 hours`) formats. Absolute time is always interpreted in the UTC timezone, not your local timezone - please convert accordingly. Supported relative date & time units: `minutes`, `hours`, `days`, `weeks`, `months`, `years`."
+        },
+        "personalData": {
+            "type": "boolean",
+            "title": "Personal data",
+            "default": true,
+            "propertyOrder": 6,
+            "description": "Enable extraction of reviewer personal data (ID, name, URL, photo) and review identifiers for advanced customer analytics. Note: Personal data processing must comply with GDPR and other applicable data protection regulations. Ensure your data governance policies permit such collection before enabling."
+        },
+        "incrementalOutput": {
+            "type": "boolean",
+            "title": "Incremental Loading",
+            "format": "checkbox",
+            "default": true,
+            "propertyOrder": 7,
+            "description": "If checked the component loads data incrementally otherwise all data from the component will be loaded."
+        },
+        "placeIdColumn": {
```

Comment:
Let's put placeIdColumn and placeUrlColumn at the beginning of config to be on top of input UI.